scripts/sct_check_dependencies.py

```diff
@@ -166,12 +166,20 @@ def main():
     # loop across python packages -- PIP
     version_requirements_pip = get_version_requirements_pip()
     for i in version_requirements_pip:
-        module = i
+        if i == "futures":
```

Comment:
The if statement can be integrated into the function `resolve_module`. The function was added recently, it will appear when master is merged into this branch